websites/serializers.py

```diff
@@ -376,6 +376,11 @@ def update(self, instance, validated_data):
             validated_data["metadata"]["file_type"] = detect_mime_type(
                 validated_data["file"]
             )
+        if "metadata" in validated_data:
+            validated_data["metadata"] = {
+                **instance.metadata,
+                **validated_data["metadata"],
+            }
         instance = super().update(
```

Comment:
Random question: This is the `update` method, which seems to be tied to HTTP PATCH requests. Is it also used for PUT? Django noob.

Reply:
I'm not sure, a quick scan of the docs didn't turn anything saying one way or the other. I would guess so?